tritium-metrics-jvm/build.gradle

```diff
@@ -19,3 +19,8 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter'
     testImplementation 'org.junit.jupiter:junit-jupiter-api'
 }
+
+moduleJvmArgs {
+    // Required for access to 'sun.net.InetAddressCachePolicy' for DNS cache metrics
+    opens 'java.base/sun.net'
+}
```

Comment:
I don't really like throwing these all over the place -- I considered adding accessors  for these parameters within https://github.com/palantir/jvm-diagnostics instead, but the existing accessors are primarily for dynamic values, where these should be static.
Do you have a preference?

Reply:
I'm ok with including this here given there's no public API or JMX accessors